### PR TITLE
254 Printing of util_transforms

### PR DESCRIPTION
**Pull request by eramdiaz**
## Description

Solves #254 

Let 'b' and `c` be:
```
@transform
class MyClass:
    def __init__(self, a):
        self.a = a
    def __call__(self, x):
         return self.a + x

a  = MyClass(4)
b = IfInfer(if_=a, else_=a)
c = Try(transform=a, catch_transform=a, exceptions=(TypeError, ZeroDivisionError), else_transform=a, finally_transform=a)
```
Before, the printing of 'b' was: 
```
IfInfer(if_=MyClass(a=4), target_mode='infer', else_=MyClass(a=4)) - "b":

   from padl.util_transforms import IfInfer
```
Now, it is:
```
IfInfer(if_=MyClass(a=4), target_mode='infer', else_=MyClass(a=4)) - "b":

   if pd_mode == infer:

      class MyClass:
          def __init__(self, a):
              self.a = a
          def __call__(self, x):
              return self.a + x

   else:

      class MyClass:
          def __init__(self, a):
              self.a = a
          def __call__(self, x):
              return self.a + x
```
Before, the printing of 'c' was:
```
Try(transform=MyClass(a=4), catch_transform=MyClass(a=4), exceptions=(<class 'TypeError'>, <class 'ZeroDivisionError'>), else_transform=MyClass(a=4), finally_transform=MyClass(a=4)) - "c":

   from padl.util_transforms import Try
```
Now, it is:
```
Try(transform=MyClass(a=4), catch_transform=MyClass(a=4), exceptions=(<class 'TypeError'>, <class 'ZeroDivisionError'>), else_transform=MyClass(a=4), finally_transform=MyClass(a=4)) - "c":

   try:

      class MyClass:
          def __init__(self, a):
              self.a = a
          def __call__(self, x):
              return self.a + x

   except (TypeError, ZeroDivisionError):

      class MyClass:
          def __init__(self, a):
              self.a = a
          def __call__(self, x):
              return self.a + x

   else:

      class MyClass:
          def __init__(self, a):
              self.a = a
          def __call__(self, x):
              return self.a + x

   finally:

      class MyClass:
          def __init__(self, a):
              self.a = a
          def __call__(self, x):
              return self.a + x
```

### CODE REVIEW

File: padl/util_transforms.py [hunk 1]
```diff
@@ -39,6 +40,14 @@ def __init__(self, if_: Transform, target_mode: Mode, else_: Optional[Transform]
         self.else_ = else_
         self.target_mode = target_mode
 
+    def _pd_longrepr(self, formatting=True, marker=None) -> str:
+        if_repr = f'if pd_mode == {self.target_mode}:\n\n' + indent(
```
Comment (wuhu) [inline padl/util_transforms.py:44]: how about putting `""`
f'if pd_mode == "{self.target_mode}":\n\n'

**Comment by jasonkhadka** [comment]
How about making `if` and `try` `except` capitalized -`If`, `Try`, `Except` like the name of transforms?

**Comment by sjrl** [comment]
> How about making `if` and `try` `except` capitalized -`If`, `Try`, `Except` like the name of transforms?

Actually I think it is okay as is

**Review by sjrl — APPROVED** [review]
Looks good!

**Comment by eramdiaz** [comment]
> > How about making `if` and `try` `except` capitalized -`If`, `Try`, `Except` like the name of transforms?
> 
> Actually I think it is okay as is

Then we leave it like it is right?

**Comment by jasonkhadka** [comment]
> > > How about making `if` and `try` `except` capitalized -`If`, `Try`, `Except` like the name of transforms?
> > 
> > 
> > Actually I think it is okay as is
> 
> Then we leave it like it is right?

yeah, was just an idea. Can leave it as it is. Looks good.

**Pull request merged**